### PR TITLE
Add Swan:Arxiv22 to pub

### PR DESCRIPTION
**Pull request by singam-sanjay**

### CODE REVIEW

**Comment by jaywonchung** [comment]
@mosharaf This PR will introduce a new `publist_topic` "Mobile Systems". Sounds right, but FL papers until know didn't have that topic. WDYT?

**Comment by mosharaf** [comment]
Remove it. We don't have that topic. Stick to the ones we have.

**Comment by mosharaf** [comment]
Use Systems + AI and Wide-Area Computing

File: source/_data/SymbioticLab.bib [hunk 1]
```diff
@@ -1026,3 +1026,28 @@ @PhDThesis{peifeng:dissertation
...
+	  title={Swan: A Neural Engine for Efficient DNN Training on Smartphone SoCs},
+	  author={Sanjay Sri Vallabh Singapuram and Fan Lai and Chuheng Hu and Mosharaf Chowdhury},
+	  archiveprefix = {arXiv},
+	  eprint= {},
+	  url= {https://arxiv.org/abs/2206.04687},
+	  year={2022},
+	  month={May},
+	  publist_confkey = {arXiv:2206.04687},
+	  publist_link = {paper || swan-arxiv22.pdf},
+	  publist_topic = {Mobile Systems},
```
Comment (jaywonchung) [inline source/_data/SymbioticLab.bib:1040]: Refer to Mosharaf's comment

**Review by jaywonchung — CHANGES_REQUESTED** [review]

File: source/_data/SymbioticLab.bib (hunk 1 above)
Comment (singam-sanjay) [reply source/_data/SymbioticLab.bib]: @jaywonchung Please check it out.

**Comment by jaywonchung** [comment]
@singam-sanjay I don't think the blogging system is smart enough to understand `publist_topic = {Systems + AI and Wide-Area Computing}`. Refer to the following:
https://github.com/SymbioticLab/SymbioticLab.github.io/blob/83bb3e5bdaa82d321a4f9afc55dd43751b4577ef/source/_data/SymbioticLab.bib#L964-L965

File: source/_data/SymbioticLab.bib (hunk 1 above)
Comment (singam-sanjay) [reply source/_data/SymbioticLab.bib]: I've separately listed the topics now.

**Comment by singam-sanjay** [comment]
I've listed the topics separately now.

**Review by jaywonchung — APPROVED** [review]
LGTM

**Pull request merged**